tests/functional/fx_ios_v1_sign_up.js

```diff
@@ -70,29 +46,6 @@ registerSuite('FxiOS v1 sign_up', {
       .then(clearBrowserState());
   },
   tests: {
-    'Fx iOS <= 6.0 sign up, verify same browser': function () {
-      return this.remote
-        .then(setupTest(UA_STRINGS['ios_firefox_6_0']))
-
-        // no screen transition, no polling even, in Fx <= 6.0
-        .then(noPageTransition(selectors.CONFIRM_SIGNUP.HEADER))
-
-        // A post-verification email should be sent, this is Sync.
-        .then(testEmailExpected(email, 1));
-    },
-
-    'Fx iOS >= 6.1 sign up, verify same browser': function () {
```

Comment:
Same as above, should we keep as `sign up, verify same browser`?

Reply:
we have `Fx iOS >= 11.0 sign up, verify same browser` below which does the same but with CWTS so we can probably avoid repeating this one